docs/developer/contributing/development-functional-tests.asciidoc

```diff
@@ -6,7 +6,7 @@ We use functional tests to make sure the {kib} UI works as expected. It replaces
 [discrete]
 === Running functional tests
 
-The `FunctionalTestRunner` is very bare bones and gets most of its functionality from its config file, located at {blob}test/functional/config.js[test/functional/config.js]. If you’re writing a plugin outside the {kib} repo, you will have your own config file.
+The `FunctionalTestRunner` is very bare bones and gets most of its functionality from its config file, located at link:{kib-repo}/tree/{branch}/test/functional/config.js[test/functional/config.js]. If you’re writing a plugin outside the {kib} repo, you will have your own config file.
```

Comment:
Doesn't appear to cause any issues to the links, but it is resulting in two slashes. I assume `kib-repo` includes a trailing slash?

https://github.com/elastic/kibana//tree/7.10/test/functional/config.js

```suggestion
The `FunctionalTestRunner` is very bare bones and gets most of its functionality from its config file, located at link:{kib-repo}tree/{branch}/test/functional/config.js[test/functional/config.js]. If you’re writing a plugin outside the {kib} repo, you will have your own config file.
```

Reply:
There are still quite a few instances with a slash before `tree` if you want to remove them. I had only done a commit suggestion on this one instance.